templates/definition/meter/sofarsolar.yaml

```diff
@@ -110,7 +110,7 @@ render: |
         {{- include "modbus" . | indent 6 }}
         register:
           address: 0x104C # BatConfig_Current_Discharge_Limit
-          type: writesingle
+          type: writemultiple
```

Comment:
Nicht `writeholdings`? Ansonsten könnte die Ergänzung in Zeile 383 der modbus.go entfallen?!

Reply:
Beides aus Gründen der Symmetrie.

Reply:
Verstehe ich nicht, tut aber nichts. Bitte ignorieren :-)